controlplane/kubeadm/internal/workload_cluster_conditions_test.go

```diff
@@ -234,39 +232,7 @@ func TestUpdateManagedEtcdConditions(t *testing.T) {
 					{Type: controlplanev1.KubeadmControlPlaneMachineEtcdMemberHealthyV1Beta2Condition, Status: metav1.ConditionUnknown, Reason: controlplanev1.KubeadmControlPlaneMachineEtcdMemberInspectionFailedV1Beta2Reason, Message: "Waiting for a Node with spec.providerID n1 to exist"},
 				},
 			},
-			expectedEtcdMembersAgreeOnMemberList:      false, // without reading members, we can not make assumptions.
-			expectedEtcdMembersAgreeOnClusterID:       false, // without reading members, we can not make assumptions.
-			expectedEtcdMembersAndMachinesAreMatching: false, // without reading members, we can not make assumptions.
-		},
-		{
-			name: "If there are provisioning machines, a node without machine should be ignored in v1beta1, reported in v1beta2 (with providerID)",
```

Comment:
Don't this test case still make sense?

Reply:
I dropped this test because it was duplicated; the other copy is stll there at [L753](https://github.com/kubernetes-sigs/cluster-api/blob/47385cdee5a66f8710d1b7263a1b3d8eeec8691d/controlplane/kubeadm/internal/workload_cluster_conditions_test.go#L753)

Reply:
The one at l.753 is for control plane components, the one here was for etcd cluster

Reply:
You are correct, good catch.
Test restored